src/common/utxobased/db/Models/ProcessorTransaction.ts

```diff
@@ -41,6 +41,7 @@ export const toEdgeTransaction = async (
       async (i: string) =>
         args.walletTools.scriptPubkeyToAddress({
           scriptPubkey: args.tx.outputs[parseInt(i)].scriptPubkey,
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Try and change this function to not be a 'one liner' and then you might be able to fix the linting problem.